### PR TITLE
[libzip] fix pc file

### PR DESCRIPTION
**Pull request by Neumann-A**

### CODE REVIEW

File: ports/libzip/use-requires.patch [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+   install(FILES ${PROJECT_BINARY_DIR}/libzip.pc DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig)
+diff --git a/libzip.pc.in b/libzip.pc.in
+index d51b0ab63..755685c03 100644
+--- a/libzip.pc.in
++++ b/libzip.pc.in
+@@ -9,6 +9,7 @@ zipcmp=${bindir}/zipcmp
+ Name: libzip
+ Description: library for handling zip archives
+ Version: @PROJECT_VERSION@
++Requires: @REQUIRES@
```
Comment (dg0yt) [inline ports/libzip/use-requires.patch:81]:
```suggestion
+Requires.private: @REQUIRES@
```

(?)

Comment (Neumann-A) [reply ports/libzip/use-requires.patch:81]: Feel free to investigate ;) I am always fine with just overlinking to make it work.

Comment (dg0yt) [reply ports/libzip/use-requires.patch:81]: Overlinking in one place may hide missing libs in another place.

But you can't deny that this PR is moving dependencies from `Libs.private` to `Requires`.

Comment (Neumann-A) [reply ports/libzip/use-requires.patch]: > may hide missing libs in another place.

I would say: That is one linking problem less for vcpkg to care about.

**Comment by Cheney-W** [comment]
```
find_package(PkgConfig REQUIRED)
pkg_check_modules(libzip REQUIRED IMPORTED_TARGET libzip)
target_link_libraries(main PkgConfig::libzip)
```
I got below error when I tested above usage in x64-windows triplet:
```
1> [CMake] -- Checking for module 'libzip'
1> [CMake] --   Package 'lzma', required by 'libzip', not found
1> [CMake] CMake Error at F:/VS2022/Common7/IDE/CommonExtensions/Microsoft/CMake/CMake/share/cmake-3.26/Modules/FindPkgConfig.cmake:607 (message):
1> [CMake]   A required package was not found
```

**Comment by Neumann-A** [comment]
@Cheney-W retry.

**Comment by Cheney-W** [comment]
Fixed.

**Review by Cheney-W — APPROVED** [review]

**Comment by Neumann-A** [comment]
@Cheney-W Forgot to tag it reviewed ?

**Pull request merged**